src/shared/components/Header/Navigation/styles.ts

```diff
@@ -1,6 +1,10 @@
 import styled from 'styled-components';
-import Colors from 'shared/colors';
 import media from 'shared/media-types';
+import Colors from 'shared/colors';
```

Comment:
why did this import change (from 2 to 3 row)? please avoid unnecessary changes to reduce merge conflicts

Reply:
Done